pkg/kubernetes/kube.go

```diff
@@ -647,3 +651,17 @@ func (kc *Client) ClusterRbacIsFound(resource, name string) error {
 	}
 	return nil
 }
+
+func (kc *Client) getWaiterInterval() time.Duration {
+	if kc.WaiterInterval > 0 {
+		return kc.WaiterInterval
+	}
+	return DefaultWaiterInterval
+}
+
+func (kc *Client) getWaiterRetries() int {
+	if kc.WaiterRetries > 0 {
```

Comment:
It would be better to be >= 0 so you can set to 0 retries.  > 0 is probably correct if WaiterRetries is number of tries.  See my other comment.

Reply:
This is actually number of tries, no retries. So setting it to 0 would not try at all. We would like to try at least once, right?